docs/3.x.x/guides/filters.md

```diff
@@ -84,54 +115,88 @@ Get the second page of results.
 
 ## Programmatic usage
 
-Requests system can be implemented in custom code sections.
+You can use `strapi-utils` to parse the query params to Strapi's standared filters programmatically if you need it.
 
 ### Extracting requests filters
 
-To extract the filters from a JavaScript object or a request, you need to call the [`strapi.utils.models.convertParams` helper](../api-reference/reference.md#strapiutils).
-
-::: note
-The returned objects are formatted according to the ORM used by the model.
-:::
-
-#### Example
-
-**Path —** `./api/user/controllers/User.js`.
+To transform the query params to Strapi's standard filters a request, you can use the `convertRestQueryParams` function from [strapi-utils](../api-reference/reference.md#strapiutils).
 
 ```js
-// Define a list of params.
-const params = {
-  '_limit': 20,
-  '_sort': 'email'
-};
+const { convertRestQueryParams } = require('strapi-utils');
 
-// Convert params.
-const formattedParams = strapi.utils.models.convertParams('user', params); // { limit: 20, sort: 'email' }
+module.exports = {
+  // Find products having a price equal or greater than `3`.
+  fetchExpensiveProducts: (params, populate) => {
+    const filters = convertRestQueryParams(params);
+
+    /**
+     *  filters = {
+     *    start: 0,
+     *    limit: 10,
+     *    sort: [{ field: 'id', order: 'desc' }],
+     *    where: [
+     *      { field: 'id', operator: 'eq', value: 1 },
+     *    ]
+     * }
+     */
+
+    // do sth with them
+  },
+};
 ```
 
-### Query usage
-
-#### Example
+### Querying data
 
-**Path —** `./api/user/controllers/User.js`.
+We added a new API to query data base on the new filters API.
 
 ```js
-module.exports = {
+const { convertRestQueryParams, buildQuery } = require('strapi-utils');
```

Comment:
When I see this, I think we should create a separate package to handle the query part... The `strapi-utils` will become a mess otherwise.

Reply:
Yes we should create a strapi-models lib or sth (to keep in mind for future refactoring)